Diagnostic/Utils/LadDiagnosticUtil.py

```diff
@@ -85,47 +94,50 @@ def generateOMIQueries(omiQueryConfiguration):
         else:
             namespace = 'root/scx'
-        if not tableName in omiQueryConfiguration:
-            omiQueryConfiguration[tableName] = defaultdict(defaultdict)
-        if not namespace in omiQueryConfiguration[tableName]:
-            omiQueryConfiguration[tableName][namespace] = defaultdict(queryParameter)
-        if not className in omiQueryConfiguration[tableName][namespace]:
-            omiQueryConfiguration[tableName][namespace][className] = queryParameter()
-            omiQueryConfiguration[tableName][namespace][className].queryCondition = condition
+        if not table_name in omi_queries:
+            omi_queries[table_name] = defaultdict(defaultdict)
+        if not namespace in omi_queries[table_name]:
+            omi_queries[table_name][namespace] = defaultdict(QueryParameter)
+        if not class_name in omi_queries[table_name][namespace]:
+            omi_queries[table_name][namespace][class_name] = QueryParameter()
+            omi_queries[table_name][namespace][class_name].queryCondition = condition
         else:
-            if omiQueryConfiguration[tableName][namespace][className].queryCondition != condition:
+            if omi_queries[table_name][namespace][class_name].queryCondition != condition:
                 raise Exception('Cannot have two different conditions on same table')
-        if not performanceCounter['counterSpecifier'] in omiQueryConfiguration[tableName][namespace][className].querySelectParameters:
-            omiQueryConfiguration[tableName][namespace][className].append(performanceCounter['counterSpecifier'])
-    return omiQueryConfiguration
+        if not performance_counter['counterSpecifier'] in omi_queries[table_name][namespace][class_name].querySelectParameters:
+            omi_queries[table_name][namespace][class_name].append(performance_counter['counterSpecifier'])
+    return omi_queries
+
 
 # Get elements from DiagnosticsMonitorConfiguration in LadCfg based on element name
 def getDiagnosticsMonitorConfigurationElement(ladCfg, elementName):
-    if ladCfg:
-        if 'diagnosticMonitorConfiguration' in ladCfg:
-            if elementName in ladCfg['diagnosticMonitorConfiguration']:
-                return ladCfg['diagnosticMonitorConfiguration'][elementName]
+    if ladCfg and 'diagnosticMonitorConfiguration' in ladCfg:
+        if elementName in ladCfg['diagnosticMonitorConfiguration']:
+            return ladCfg['diagnosticMonitorConfiguration'][elementName]
+    return None
+
 
 # Get fileCfg form FileLogs in LadCfg
 def getFileCfgFromLadCfg(ladCfg):
     fileLogs = getDiagnosticsMonitorConfigurationElement(ladCfg, 'fileLogs')
-    if fileLogs:
-        if 'fileLogConfiguration' in fileLogs:
-            return fileLogs['fileLogConfiguration']
+    if fileLogs and 'fileLogConfiguration' in fileLogs:
+        return fileLogs['fileLogConfiguration']
+    return None
+
 
 # Generates OMI queries from LadCfg
 def generatePerformanceCounterConfigurationFromLadCfg(ladCfg):
```

Comment:
As far as I understand, the functions here are not used/exercised in production (because Portal never sets "ladCfg" in the protected settings), so I think it'd be great if we have some unit tests on the functions here. By the way, I thought AI team left some test code for these features, but I couldn't find them. Maybe they checked them in on another repo? I'll look. Thanks.